boskos/boskos.go

```diff
@@ -0,0 +1,295 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var configPath = flag.String("config", "projects.json", "Path to init projects file")
+var storage = flag.String("storage", "", "Path to presistent volume to load the state")
+
+type GCPProject struct {
```

Comment:
This could lease out cluster names, disks, networks, etc just as easily as projects. There is nothing about `name string` that binds this to leasing out projects rather than some other resource.